packages/fxa-settings/src/index.tsx

```diff
@@ -62,7 +63,9 @@ try {
     <React.StrictMode>
       <ApolloProvider client={apolloClient}>
         <AuthContext.Provider value={{ auth: authClient }}>
-          <App {...{ queryParams }} />
+          <AppErrorBoundary>
```

Comment:
I'm not entirely sure where this should live FWIW, in payments we wrap `StripeProvider` and `ReduxProvider` but not `AppContext.Provider` or `AppLocalizationProvider`...

Reply:
we'll figure it out eventually ;)